code/modules/modular_computers/file_system/picture_file.dm

```diff
@@ -0,0 +1,39 @@
+/**
+ * PNG file type
+ * Stores a picture to be used for programs.
+ */
+
+GLOBAL_VAR_INIT(next_ntos_picture_uid, 0)
+
+/proc/get_next_ntos_picture_path()
+	return "ntos_picture_[GLOB.next_ntos_picture_uid++].png"
+
+/datum/computer_file/picture
+	filetype = "PNG" // the superior filetype
+	size = 2
+	var/datum/picture/stored_picture
+	var/picture_name
```

Comment:
Are these used outside this file? If so they must be documented, same with the type path